pkg/apis/events/v1alpha1/test_helper.go

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
I'm not very convinced of this test helper. I think it will be packaged along with non-test code due to its name (not ending in *_test.go)
But most importantly, we've been using the Option pattern for tests, for example, look at the reconciler tests https://github.com/google/knative-gcp/blob/master/pkg/reconciler/testing/pullsubscription.go...
I think we might need to move those out of pkg/reconciler, and put them in pkg/testing, and then you won't depend on anything within reconciler... But for now, I think you can just create these 3 functions as private helpers in the go file of the UT that uses them... I see only being used in a single place?